src/app/src/main/kotlin/org/imperial/mrc/hint/exceptions/HintExceptionHandler.kt

```diff
@@ -106,19 +106,12 @@ class HintExceptionHandler(private val errorCodeGenerator: ErrorCodeGenerator,
         val messageArguments = arrayOf(
                 appProperties.applicationTitle,
                 errorCodeGenerator.newCode(),
-                appProperties.supportEmail
+                appProperties.supportEmail,
+                if (originalMessage != null) " $originalMessage" else ""
```

Comment:
```suggestion
             originalMessage
```

Reply:
There's really no need for this check.

Reply:
Without this won't the formatter render this as `null` instead of an empty string? I tried this here https://pl.kotl.in/m_KcGN0mW

Reply:
> Without this won't the formatter render this as `null` instead of an empty string? I tried this here https://pl.kotl.in/m_KcGN0mW

```
private fun unexpectedError(status: HttpStatus,
                                request: HttpServletRequest,
                                originalMessage: String? = ""): ResponseEntity<Any>

instead of 
private fun unexpectedError(status: HttpStatus,
                                request: HttpServletRequest,
                                originalMessage: String? = null"): ResponseEntity<Any>
    {
    

```

Reply:
Ah yeah makes sense to default to empty string, but the `originalMessage` can still come in as `null` right? Exception type can have a `null` message. I don't think we can change to
```
originalMessage: String = ""
```
for example?